base/App/Home/HomeScreen.swift

```diff
@@ -29,6 +29,20 @@ struct HomeScreen: View {
             )
         }
         .navigationTitle("Hello, \(settings.user?.username ?? "World")!")
+        .toolbar {
+            Button(
+                action: {
+                    Navigation.path.modal {
+                        NavigationStack {
+                            AcknowledgmentView()
+                        }
+                    }
+                },
+                label: {
+                    Image(systemName: "info.circle")
+                }
+            )
+        }
```

Comment:
Only thing to note... upon clicking this button the inital time I got the following message in console. It did not effect the use of the app aside from a slight delay in opening on this time. Every subsequent time was fine. I have seen this before and am unsure what its related to.

> Update NavigationAuthority bound path tried to update multiple times per frame.